doc/source/readers.rst

```diff
@@ -141,46 +141,85 @@ time etc. The following attributes are standardized across all readers:
+
+In addition to the generic ``start_time`` and ``end_time`` pieces of metadata
+there are other time fields that may be provided if the reader supports them.
+These items include:
+
+* ``observation_start_time``: The point in time when a sensor began recording
+  for the current data.
+* ``observation_end_time``: Same as ``observation_start_time``, but when data
+  has stopped being recorded.
+* ``scheduled_start_time``: The "human friendly" time describing the start of
+  the data observation interval or repeat cycle. This time is often on a round
+  minute (seconds=0). Along with the scheduled end time, these times define the
+  regular interval of the data collection. For example, GOES-16 ABI full disk
+  images are collected every 10 minutes (in the common configuration) so
+  ``scheduled_start_time`` and ``scheduled_end_time`` would be 10 minutes apart
+  regardless of when the instrument recorded data inside that interval.
+* ``scheduled_end_time``: Same as ``scheduled_start_time``, but the end of the
+  interval.
+
+In general, ``start_time`` and ``end_time`` will be set to the "scheduled"
+time by the reader. This ensures that other Satpy components get a
+consistent time for calculations (ex. generation of solar zenith angles)
+and can be reused between bands.
+
+.. _orbital_parameters:
+
+Orbital Parameters
+------------------
+
+Orbital parameters describe the position of the satellite. As such they
+typically come in a few "flavors" for the common types of orbits a satellite
+may have.
+
+For *geostationary* satellites it is described using the following scalar attributes:
+
+  * ``satellite_actual_longitude/latitude/altitude``: Current position of the satellite at the
+    time of observation in geodetic coordinates (i.e. altitude is relative and normal to the
+    surface of the ellipsoid).
+  * ``satellite_nominal_longitude/latitude/altitude``: Center of the station keeping box (a
+    confined area in which the satellite is actively maintained in using maneuvers). Inbetween
+    major maneuvers, when the satellite is permanently moved, the nominal position is constant.
+  * ``nadir_longitude/latitude``: Intersection of the instrument's Nadir with the surface of the
+    earth. May differ from the actual satellite position, if the instrument is pointing slightly
+    off the axis (satellite, earth-center). If available, this should be used to compute viewing
+    angles etc. Otherwise, use the actual satellite position.
+  * ``projection_longitude/latitude/altitude``: Projection center of the re-projected data. This
+    should be used to compute lat/lon coordinates. Note that the projection center can differ
+    considerably from the actual satellite position. For example MSG-1 was at times positioned
+    at 3.4 degrees west, while the image data was re-projected to 0 degrees.
+  * [DEPRECATED] ``satellite_longitude/latitude/altitude``: Current position of the satellite at
+    the time of observation in geodetic coordinates.
```

Comment:
I think this is dependent on the reader. For example, I see some readers used the `satellite_` with the projection coordinates. Others I think were using the actual position. I think that's why @sfinkens did all the work on `orbital_parameters` (because of the inconsistencies).

Reply:
Keep in mind this documentation was here before, I just moved it. Also, I plan on making another PR removing these deprecated keys anyway so there is no need to document what they changed to. The orbital_parameters for these have existed for a long time.

Reply:
Ok